pkg/runtime/docker/client.go

```diff
@@ -105,8 +106,20 @@ func (dc *dockerClient) AttachContainer(container string) (err error) {
 func (dc *dockerClient) RunContainer(image string, config *runtime.ContainerConfig, name string) (string, error) {
 	portBindings := make(nat.PortMap)
```

Comment:
factor this out to a `portBindingsToPortMap` function please?

Reply:
Factored in a182160.